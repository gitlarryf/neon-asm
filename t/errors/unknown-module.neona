
PUSHN 5
PUSHS "*"
CALLMF string.repeat
CALLP print
RET
