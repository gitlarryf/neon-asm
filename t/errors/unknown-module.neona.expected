Unknown module, or module not imported.  Line 3
